### PR TITLE
Create app form models

### PR DESCRIPTION
**Pull request by Serg-Bartosh**

### CODE REVIEW

File: forms/models.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
 from django.db import models
 
-# Create your models here.
+
+class Form(models.Model):
+    name = models.CharField(max_length=255, help_text='Name form', null=False)
+    descriptions = models.CharField(max_length=255, help_text='Description form')
```
Comment (s373r) [inline forms/models.py:6]: please name it in single form: `description`

Comment (s373r) [inline forms/models.py:6]: I guess `help_text` can be deleted

File: forms/admin.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
 from django.contrib import admin
```
Comment (s373r) [inline forms/admin.py:2]: remove a blank line between imports

File: forms/admin.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
...
 
-# Register your models here.
+from forms.models import Form, FormAnswer
```
Comment (s373r) [inline forms/admin.py:3]: please use a relative import

**Review by s373r — CHANGES_REQUESTED** [review]

File: forms/admin.py (hunk 2 above)
Comment (Serg-Bartosh) [reply forms/admin.py:2]: We need this line!!!

File: forms/models.py (hunk 1 above)
Comment (Serg-Bartosh) [reply forms/models.py]: OK

Comment (Serg-Bartosh) [reply forms/models.py]: YES SIR

**Review by s373r — APPROVED** [review]
Looks cool!

**Pull request merged**